### DISCUSSION TITLE
Option to hide the “filter folder” field

### DISCUSSION THREAD

**Discussion by phiw13** [ViennaRSS/vienna-rss#582] [Ideas]
Would be nice to have the option to hide  “filter folder” field – an additional menu item in the “View” menu probably.

**Comment by JanX2** [comment]
@josh64x2 What about doing this the same way that the articles filter (bottom-right button) is done: hide by default and show after a click on a filter icon added to the lower left-hand corner of the main window?

**Comment by squintgit** [comment]
+1 - thanks in advance for working on this!

**Comment by josh64x2** [comment]
This is going to be difficult due to the way the `AppController` is really a giant WindowController and ViewController with no proper view management of view hierarchy.

**Comment by josh64x2** [comment]
Just an update on this - once #761 has been merged in we can probably start working on this 😄

**Comment by josh64x2** [comment]
@Eitot I remembered we were going to look at using `NSStackView` after we made 10.9 the minimum. `NSStackView` would help with this feature request - do you think now is the time to implement or are there other UI changes we should do first?

**Comment by Eitot** [comment]
I have a branch somewhere, but I found that PSMTabBarControl works not better with stack views. I am holding off until I have a look at MMTabBarView.

The sidebar has a low priority for me. However, it should be easy to implement this feature request using NSScrollViews’ `addFloatingSubview` method.

**Comment by squintgit** [comment]
Still looking forward to this enhancement. I only have 5 folders and 10 subscriptions so I have never used that field.

**Comment by barijaona** [comment]
Text filters have now been grouped into a single combo search field
